library/src/main/java/com/chuckerteam/chucker/api/ChuckerCollector.kt

```diff
@@ -48,8 +63,10 @@ class ChuckerCollector @JvmOverloads constructor(
      * @param transaction The HTTP transaction sent
      */
     internal fun onRequestSent(transaction: HttpTransaction) {
+        if (!httpFeature.enabled) return
```

Comment:
I suggest to add braces here for readability and to match other code lines, like below where condition with only one line in it has braces.

Reply:
I use braces when there is something to do.
My point of view is this is an early return, in order to do nothing. If I do nothing I have no braces 🤷‍♂ 

What do you think of that?

Reply:
> I use braces when there is something to do.

Agree with @olivierperez 
Also the style guide here https://developer.android.com/kotlin/style-guide#braces allows single-line if statements like:

```kotlin
if (condition) effect
```